README.md

```diff
@@ -7,7 +7,7 @@ WordPress feature plugin to bring Progressive Web App (PWA) capabilities to Core
 **Contributors:** [google](https://profiles.wordpress.org/google), [xwp](https://profiles.wordpress.org/xwp), [westonruter](https://profiles.wordpress.org/westonruter), [albertomedina](https://profiles.wordpress.org/albertomedina)  
 **Tags:** [pwa](https://wordpress.org/plugins/tags/pwa), [progressive web apps](https://wordpress.org/plugins/tags/progressive-web-apps), [service workers](https://wordpress.org/plugins/tags/service-workers), [web app manifest](https://wordpress.org/plugins/tags/web-app-manifest)  
 **Requires at least:** 5.5  
-**Tested up to:** 5.7  
+**Tested up to:** 5.8.2
```

Comment:
```suggestion
**Tested up to:** 5.8
```

Reply:
Since WordPress automatically bumps the patch version for you.

Reply:
Also note that this was done in the SVN tag already: https://plugins.trac.wordpress.org/browser/pwa/tags/0.6.0/readme.txt#L5